### PR TITLE
Promote the win_hotfix Module from community

### PR DESCRIPTION
**Pull request by Yaish25491**
##### SUMMARY
Promote the win_hotfix Module from community
Updating documentation examples.

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME

plugins/modules/win_hotfix.ps1
plugins/modules/win_hotfix.py
tests/integration/targets/win_hotfix/aliases
tests/integration/targets/win_hotfix/defaults/main.yml
tests/integration/targets/win_hotfix/tasks/main.yml
tests/integration/targets/win_hotfix/tasks/tests.yml
tests/integration/targets/win_hotfix/tasks/tests_2012R2.yml

### CODE REVIEW

File: tests/integration/targets/win_hotfix/aliases [hunk 1]
```diff
@@ -0,0 +1,2 @@
+shippable/windows/group4
```
Comment (jborean93) [inline tests/integration/targets/win_hotfix/aliases:1]: While the test is unstable so won't run we should make sure this is in a valid group

```suggestion
shippable/windows/group2
```

File: plugins/modules/win_hotfix.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  type: str
+  sample: KB3172729
+kbs:
+  description: The KB for each hotfix in the msu,
+  returned: success
+  type: list
+  elements: str
+  sample:
+  - KB3172729
+  version_added: '1.10.0'
```
Comment (jborean93) [inline plugins/modules/win_hotfix.py:144]:
```suggestion
```

File: plugins/modules/win_hotfix.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  type: str
+  sample: Package_for_KB3172729~31bf3856ad364e35~amd64~~6.3.1.0
+identifiers:
+  description: The DISM identifiers for each hotfix in the msu.
+  returned: success
+  type: list
+  elements: str
+  sample:
+  - Package_for_KB3172729~31bf3856ad364e35~amd64~~6.3.1.0
+  version_added: '1.10.0'
```
Comment (jborean93) [inline plugins/modules/win_hotfix.py:131]:
```suggestion
```

**Review by jborean93 — APPROVED** [review]

**Comment by Yaish25491** [comment]
branch rebased and changes added

File: plugins/modules/win_hotfix.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_hotfix
+short_description: Install and uninstalls Windows hotfixes
+description:
+- Install, uninstall a Windows hotfix.
```
Comment (jborean93) [inline plugins/modules/win_hotfix.py:11]: Needs the version added entry.

File: plugins/modules/win_hotfix.py [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+  community.windows.win_hotfix:
+    hotfix_identifier: Package_for_KB3172729~31bf3856ad364e35~amd64~~6.3.1.0
+    state: absent
+  register: hotfix_uninstall
+
+- ansible.windows.win_reboot:
+  when: hotfix_uninstall.reboot_required
+
+- name: Uninstall hotfix with KB (not recommended)
+  community.windows.win_hotfix:
```
Comment (jborean93) [inline plugins/modules/win_hotfix.py:108]: Needs to change from `community.windows.win_hotfix` to `ansible.windows.win_hotfix`.

**Comment by Yaish25491** [comment]
Changes were added again after being lost while rebasing

**Review by jborean93 — APPROVED** [review]

**Pull request merged**